views.py

```diff
@@ -148,33 +148,20 @@ def confirm_registration(request):
     form = RegistrationConfirmationForm(user=user, token=token, token_generator=tokenGenerator, initial={'userid': uid, 'authtoken': token, 'username': user.username if user else ''})
   return tracker_response(request, 'tracker/confirm_registration.html', {'formuser': user, 'tokenmatches': tokenGenerator.check_token(user, token) if token else False, 'form': form, 'csrftoken': get_csrf_token(request)})
 
-def tracker_response(request=None, template='tracker/index.html', qdict={}, status=200):
+def tracker_response(request=None, template='tracker/index.html', qdict=None, status=200):
```

Comment:
Yeah, the default arg is only initialized the one time the old way, so if somebody modifies qdict, every other default call will get the modified one... which is bad, for obvious reasons.